docs/mobile/events.md

```diff
@@ -8,12 +8,27 @@ events on mobile platforms. All mobile events MUST use a namespace of
 
 <!-- toc -->
 
+- [Application unresponsive UI](#application-unresponsive-ui)
 - [Lifecycle instrumentation](#lifecycle-instrumentation)
   * [iOS](#ios)
   * [Android](#android)
 
 <!-- tocstop -->
 
+## Application unresponsive UI
+
+This event denotes when an application's UI "freezes", making users unable to interact with the app. It happens when the
+main thread (which is in charge of rendering the UI) gets blocked by a long-running task for a certain amount of time.
```

Comment:
What do you think about being less descriptive about the reasons behind the main thread not being responsive given how this could differ between platforms? There could be other reasons, like thermal throttling, core restrictions imposed by the OS, etc.

Reply:
Sure we can make it less specific.